### PR TITLE
Section-3 55-58

### PR DESCRIPTION
**Pull request by seanho96**
- https://trello.com/c/uOucEvJg
- Display hello world through basic MVC pattern
- Add prettier plugin

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+    def home
+    end
```
Comment (tlkiong) [inline app/controllers/pages_controller.rb]: 2 spaces indent pls

**Review by tlkiong — CHANGES_REQUESTED** [review]

**Comment by seanho96** [comment]
The prettier plugin here will be overridden by Rubocop in PR #3.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "rubySingleQuote": false,
```
Comment (tlkiong) [inline .prettierrc]: Indentation?

**Review by tlkiong — CHANGES_REQUESTED** [review]
Pay attention to details.
Always double, triple check your work.

> To create a masterpiece, you must first care for the details

File: .prettierrc (hunk 2 above)
Comment (seanho96) [reply .prettierrc]: My bad, overlooked this 😓

**Review by tlkiong — APPROVED** [review]

**Pull request merged**